workloads/router-perf-v2/env.sh

```diff
@@ -21,6 +21,16 @@ export NUMBER_OF_ROUTERS=${NUMBER_OF_ROUTERS:-2}
 export HOST_NETWORK=${HOST_NETWORK:-true}
 export NODE_SELECTOR=${NODE_SELECTOR:-'{node-role.kubernetes.io/workload: }'}
 
+# Cluster information
+export CLUSTER_ID=$(oc get clusterversion -o jsonpath='{.items[].spec.clusterID}')
+export CLUSTER_NAME=$(oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}')
+export OPENSHIFT_VERSION=$(oc version -o json |  jq -r '.openshiftVersion')
+KUBERNETES_MAJOR_VERSION=$(oc version -o json |  jq -r '.serverVersion.major')
+KUBERNETES_MINOR_VERSION=$(oc version -o json |  jq -r '.serverVersion.minor')
+export KUBERNETES_VERSION=${KUBERNETES_MAJOR_VERSION}.${KUBERNETES_MINOR_VERSION}
+export CLUSTER_NETWORK_TYPE=$(oc get network.config/cluster -o jsonpath='{.spec.networkType}')
+export CLOUD_TYPE=$(oc get infrastructure cluster -o jsonpath='{.status.platformStatus.type}')
```

Comment:
We need to go deeper on the cloud type. Both AWS and ROSA show AWS as platform on the platformStatus.type.

ROSA clusters have additional tags:

```
 "platformStatus": {
            "aws": {
                "region": "us-west-2",
                "resourceTags": [
                    {
                        "key": "red-hat-clustertype",
                        "value": "rosa"
                    },
                    {
                        "key": "red-hat-managed",
                        "value": "true"
                    }
                ]
            },
            "type": "AWS"
        }
```

Reply:
This can be used also here

https://github.com/cloud-bulldozer/e2e-benchmarking/pull/333/files#diff-00ca231aa758364161a511f7fe21d6ac3c3e7547aab778b04b6061b39c5612e6R129